<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="registerFaceButton">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Register Face</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadFaceButton">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Load Face</string>
      </property>
     </widget>
     <widget class="QPushButton" name="identifyFaceButton">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Identify Face</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quitButton">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="facelabel">
      <property name="maximumSize">
       <size>
        <width>1080</width>
        <height>1080</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>720</width>
        <height>360</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="responsefacelabel">
      <property name="maximumSize">
       <size>
        <width>1080</width>
        <height>1080</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>720</width>
        <height>360</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
